src/web/mjs/connectors/AstralLibrary.mjs

```diff
@@ -10,5 +10,28 @@ export default class AstralLibrary extends WordPressMadaraNovel {
         this.url = 'https://astrallibrary.net';
 
         this.novelObstaclesQuery = 'div.ad';
+        this.queryMangas = 'div.item-summary';
+    }
+
+    async _getMangasFromPage(page) {
+        let request = new Request(new URL('page/'+page+1,'https://www.astrallibrary.net/manga-tag/manga/'));
+        let data = await this.fetchDOM(request, this.queryMangas);
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element.querySelector('.list-chapter a'), request.url),
```

Comment:
This will get a link to the latest chapter instead of the manga and may interfere with bookmarked mangas (bookmarks are not detected, bookmarks will never find new chapters, ...).

Reply:
Obsolete since this method should not be overwritten